development-vm/README.md

```diff
@@ -1,276 +1,226 @@
-# Developing on GOV.UK Virtual Machines
+# Getting Started
 
-Welcome to GOV.UK. The following instructions show you how to get your
-development environment running.
+Welcome to GOV.UK.
 
-## 0. Context
+Our development environment is an Ubuntu virtual machine (VM). We aim to achieve [dev-prod parity](http://www.12factor.net/dev-prod-parity).
```

Comment:
Do we need this paragraph? Feels more contextual than instructional. Just feels a bit out of place when I read from the beginning...